src/test/java/de/eposcat/master/generators/BigAttributesCaseSetup.java

```diff
@@ -1,103 +1,94 @@
+import de.eposcat.master.generators.data.FillerAttributesStats;
+import de.eposcat.master.generators.data.PerformanceTestAttribute;
+import de.eposcat.master.model.Attribute;
+import de.eposcat.master.model.Page;
 
 /**
  * The databases of this class get filled with data when first run and then will be reused for the tests
  * To generate new start data, delete the running containers (it seems like you cannot name the containers with testcontainers, so look for containers based on the images)
- *
+ * <p>
  * !!!!!!!
  * Currently there can be only one running oracle container, so you might need to stop the one from the integration tests or the other Test cases
  * There is probably an issue with the ports and this behavior might be fixed in a future update.
  * It is possible though to pause and resume the containers manually and save yourself some time!
  * !!!!!!!
- *
+ * <p>
  * This class should not be run as part of the Unit and Integration Tests
  */
 @Testcontainers
-public class BigAttributesCaseSetup extends PerformanceTestContainerStartup{
+public class BigAttributesCaseSetup extends PerformanceTestContainerStartup {
 
     private static final Logger log = LoggerFactory.getLogger(BigAttributesCaseSetup.class);
 
-    static {
-        setupContainers("cornerCase");
-        startup();
-    }
-
-    static public void setupCornerCaseScenario(){
-        numberOfStartEntities = 1000;
-
-        maxNumberOfAttributes = 10;
-        meanNumberOfAttributes = 5;
-        numberOfStartAttributes = 30;
-
-        queryAttributes = new ArrayList<>(Arrays.asList(
-                new PerformanceTestAttribute("bigAttribute", 5d, () -> generateBigString(1000)),
-                new PerformanceTestAttribute("otherAttribute", 50d, () -> String.valueOf(r.nextBoolean()))
-        ));
-    }
-
-    static public String generateBigString(int sizeInKB){
-        byte[] bytes = new byte[sizeInKB * 1000];
-        r.nextBytes(bytes);
-        return Arrays.toString(bytes);
-    }
-
-    static void startup(){
-        setupCornerCaseScenario();
-        setupData();
+    private static final String TEST_CASE_NAME = "Big Attribute";
+    private static final int NUMBER_OF_START_ENTITIES = 100;
+    private static final int NUMBER_OF_START_ATTRIBUTES = 30;
+    private static final int MEAN_NUMBER_OF_ATTRIBUTES = 5;
+    private static final int MAX_NUMBER_OF_ATTRIBUTES = 10;
+    private static final List<PerformanceTestAttribute> QUERY_ATTRIBUTES = Arrays.asList(
+            new PerformanceTestAttribute("bigAttribute", 5d, () -> generateBigString(1000)),
+            new PerformanceTestAttribute("otherAttribute", 50d, () -> String.valueOf(RANDOM_GEN.nextBoolean()))
+    );
+
+    @BeforeAll
+    static void setupData() {
+        assertThat(POSTGRES.isRunning(), is(true));
+        log.info("Postgres container is running. Generating initial data for JSON and EAV approach.");
+
+        initAdapters();
```

Comment:
Shall we check for emptyDBs? Similar to CommonCaseSetup ` // generate test data only for empty DBs`

Reply:
Yes I'll update it. I'm also working on adding the queries in the common case back again too. I will only run the 50% for names and 25% for values cases, instead of all 6.